jobs/build-cosa.Jenkinsfile

```diff
@@ -26,7 +26,8 @@ properties([
          silentResponse: false,
          regexpFilterText: '$COREOS_ASSEMBLER_GIT_REF',
          regexpFilterExpression: 'main|rhcos-.*'
-        ]
+        ], 
+        cron('H H * * 1,3')
```

Comment:
Sorry for a silly question, but isn't cron format ('M H * * 1,3') ?

Reply:
Ok, sorry for the noise. I looked it up and in Jenkins it specifies a range.

/lgtm